freer-extras/src/Control/Monad/Freer/Extras/Beam.hs

```diff
@@ -157,10 +157,12 @@ data BeamEffect r where
     => SqlSelect Sqlite a
     -> BeamEffect (Maybe a)
 
-  SqlCommand :: Sqlite.Query -> BeamEffect ()
-  -- ^ mainly to execute sql commands in between sql queries
-  -- e.g., DROP INDEX AND CREATE INDEX before insertions, delete or updates
-  -- This is necessary on large DB
+  SqlCommand ::
+    Sqlite.Query
+    -> BeamEffect ()
+    -- ^ mainly to execute sql commands in between sql queries
+    -- e.g., DROP INDEX AND CREATE INDEX before insertions, delete or updates
+    -- This is necessary on large DB
```

Comment:
Given that this is a constructor in the GADT, should we instead do this? Cause I think the doc comment is for the whole constructor instead of the argument or return value, maybe @etiennejf can confirm this
```suggestion
  -- | mainly to execute sql commands in between sql queries
  -- e.g., DROP INDEX AND CREATE INDEX before insertions, delete or updates
  -- This is necessary on large DB
  SqlCommand :: Sqlite.Query -> BeamEffect ()
```

Reply:
Yep I agree the comment is for the whole constructor. Sorry for the late review. I've missed this PR.

Reply:
Thanks! I've updated the comment as suggested